.taskcluster.yml

```diff
@@ -16,7 +17,7 @@ tasks:
           - /bin/bash
           - '-c'
           - >-
-            git clone ${event.pull_request.head.repo.git_url} taskcluster-worker-runner &&
+            git clone ${event.pull_request.head.repo.clone_url} taskcluster-worker-runner &&
```

Comment:
funny name for a folder `taskcluster-worker-runner` :) surely not a copy-paste?

Reply:
Lol I saw this too.. Can rename